### PR TITLE
Mobile fixes

### PR DESCRIPTION
**Pull request by beamalsky**
Fixes for Issue #273 and improved nav bar scaling. The new mobile Current Events view looks like they wanted, but I'm not sure the changes I made were the most straightforward.

screen shot 2018-03-27 at 4 49 46 pm

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-psycopg2==2.6.2
+psycopg2==2.7.1
```
Comment (reginafcompton) [inline requirements.txt]: You recently merged this change to master: you should not have to duplicate it here! Try to keep your local master up-to-date with the remote master, especially before checking out a new branch. Let me know if you have questions about this.

File: lametro/static/css/city_custom.css [hunk 2]
```diff
@@ -258,7 +262,7 @@ div#toggleControls { margin-bottom: 1em; }
     }
 }
 
-@media (min-width: 767px) and (max-width: 991px) {
+@media (min-width: 767px) and (max-width: 1264px) {
```
Comment (reginafcompton) [inline lametro/static/css/city_custom.css]: Nice catch!

File: lametro/models.py [hunk 3]
```diff
@@ -191,6 +191,12 @@ def upcoming_board_meeting(cls):
 
     @classmethod
     def current_meeting(cls):
+        
+        #use this line to hardcode one current event
+        #return Event.objects.filter(start_time='2017-06-15 13:30:00-05')
+        #use this line to hardcode two current events
+        #return Event.objects.filter(start_time='2017-11-30 11:00:00-06')
```
Comment (reginafcompton) [inline lametro/models.py]: I'd actually like to see these comments consolidated with the "docstring" (lines 201 ff.)....something, like, "To hardcode current event(s) for testing, then return these querysets...." (etc etc etc).

File: requirements.txt (hunk 1 above)
Comment (beamalsky) [reply requirements.txt]: 👌🏻

**Pull request merged**